operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -109,7 +109,7 @@ spec:
         runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/WebAppServicesHighCPUUsage.md
         summary: WebApp Services excessive CPU usage.
         description: Pod {{ $labels.pod }} is consumming too much CPU. Please investigate.
-        dashboard_url: https://grafana.gitpod.io/d/6581e46e4e5c7ba40a07646395ef7b23/kubernetes-compute-resources-pod?var-cluster={{ $labels.cluster }}&var-namespace=default&var-pod={{ $labels.pod }}
+        dashboard_url: https://grafana.gitpod.io/d/gitpod-services/meta-services?orgId=1&var-datasource=VictoriaMetrics&var-cluster={{ $labels.cluster }}&var-node=All&var-component=All&from=now-1h&to=now
```

Comment:
The old link gets you into a generic dashboard that in this case does not show the same data as the alert.
We could try to fix this, but I don't see why: The alert was created with that Meta Service dashboard in mind, uses the same query, and immediately allows you to see what's going on.

The only "downside" is that we can't use a pod-specific selector. But I think it's ok bc you either immediately see the outlier, or can so select quickly by hand.

Reply:
Maybe there's another way. I think CPU increased usage for a particular pod is rather uninteresting. What we actually care in the alert is if the overall average has reached a dangerous threshold - but a single pod spiking isn't such a big deal. For that, we could use the built-in `Cluster Resources` dashboard and point against the deployment.

This would solve 2 things:
* The alert being trigger happy (because it's per pod)
* Re-using an existing dashboard for this